src/py/aspen/config/production.py

```diff
@@ -1,12 +1,10 @@
-import uuid
-
 from aspen.config import config
 
 
-class ProductionConfig(config.Config, descriptive_name="prod"):
+class ProductionConfig(config.RemoteConfig, descriptive_name="prod"):
     @config.flaskproperty
-    def SECRET_KEY(self) -> str:
-        return uuid.uuid4().hex
```

Comment:
do we need to add `SECRET_KEY` to the `RemoteConfig` if we're removing it here?

Reply:
ok i see you added this back in later PRs, nevermind

Reply:
A terrible implementation is provided by the base class now.